### DISCUSSION TITLE
Non-emoji Unicode characters not showing up

### DISCUSSION THREAD

**Discussion by poperigby** [fdw/rofimoji#203] [Q&A] [answered]
Sorry if this is a dumb question, but I'm really struggling with this. I'm using the [nixpkgs version of rofimoji](https://github.com/NixOS/nixpkgs/blob/nixos-24.05/pkgs/by-name/ro/rofimoji/package.nix), and non-emoji Unicode characters aren't showing up. I've tried searching things like "pi" and "degrees", but only emojis come up. Am I doing something wrong?

**Comment by fdw** [comment, marked as answer]
No worries 🙂

Have you defined any [`files`](https://github.com/fdw/rofimoji?tab=readme-ov-file#options), either with the parameter or the config?

**Reply by poperigby** [reply]
There we go. Thanks :slightly_smiling_face:.
